app/routes/wallet/credentials.py

```diff
@@ -1,34 +1,59 @@
-from typing import Optional
+from typing import List, Optional
 
 from aries_cloudcontroller import (
     AttributeMimeTypesResult,
-    CredInfoList,
     CredRevokedResult,
-    IndyCredInfo,
-    VCRecord,
-    VCRecordList,
     W3CCredentialsListRequest,
 )
-from fastapi import APIRouter, Depends
+from fastapi import APIRouter, Depends, Query
 
 from app.dependencies.acapy_clients import client_from_auth
 from app.dependencies.auth import AcaPyAuth, acapy_auth_from_header
 from app.exceptions import handle_acapy_call
+from app.models.wallet import CredInfoList, IndyCredInfo, VCRecord, VCRecordList
 from shared.log_config import get_logger
 
 logger = get_logger(__name__)
 
 router = APIRouter(prefix="/v1/wallet/credentials", tags=["wallet"])
 
 
-@router.get("", response_model=CredInfoList)
+@router.get(
+    "",
+    response_model=CredInfoList,
+    summary="Fetch a list of credentials from the wallet",
+)
 async def list_credentials(
     count: Optional[str] = None,
     start: Optional[str] = None,
```

Comment:
We can rename this limit / offset, and just pass it as the count / start arguments.
I have an open PR with acapy to rename these, but we can rename it on our side so long. And they should be Optional[int]